find_modules.pl

```diff
@@ -9,13 +9,13 @@
     ;  !,
        fail
     ).
-find_module_declaration(Handle, [(ModuleName, ModuleVersion)]) :-
+find_module_declaration_worker(Handle, ModuleName) :-
```

Comment:
Worker makes it sound like it's a separate thread to me. I see what you're going for I think though. No need to change, just fyi